firebolt.go

```diff
@@ -90,13 +90,15 @@ func (dialector Dialector) DataTypeOf(field *schema.Field) string {
 	case schema.Bool:
 		return "BOOLEAN"
 	case schema.Int, schema.Uint:
-		return "INT"
+		return "LONG"
 	case schema.Float:
-		return "FLOAT"
+		return "DOUBLE"
```

Comment:
This affects precision. Are we sure Ironsource are happy with this change? Though I guess we're increasing precision so it shouldn't be an issue.

Reply:
I think since we convert it to go float64 it should be fine. Yet we're not sure what type they've used before